### PR TITLE
Add `c:animal_foods` tag

### LINKED ISSUE #4036: A tag to group up all animal food tags
A user made a pr here: https://github.com/neoforged/NeoForge/pull/1465

still discussing the tag name at the moment. The idea is some people may want to collect up all foods that can be fed to mobs.

### PR DESCRIPTION
**Pull request by TelepathicGrunt**
A tag to collect all the new mojang food tags into one parent tag so modders can use this to validate if an item is a feed item for some unknown animal. May make sorting easier or allow stocking up animal feeding machines without non-feedable items slipping in.  Railcraft Reborn had a specific use case for this (see Neo PR)

Closes: https://github.com/FabricMC/fabric/issues/4036

Neoforge PR: https://github.com/neoforged/NeoForge/pull/1465

### CODE REVIEW

**Review by modmuss50 — COMMENTED** [review]
Change is good, the build is failing as one of the resource files isnt matching, please try re-running the datagen.

**Comment by TelepathicGrunt** [comment]
@modmuss50 I ran the datagen for the convention tags v2 module and no change. I don't see what specific resource is not matching
image
image

**Comment by modmuss50** [comment]
Ill take a look, I should improve the github actions to actually show the diff...

File: fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json [hunk 1]
```diff
@@ -168,6 +168,7 @@
   "tag.item.c.ender_pearls": "Ender Pearls",
   "tag.item.c.fertilizers": "Fertilizers",
   "tag.item.c.foods": "Foods",
+  "tag.item.c.animal_foods": "Animal Foods",
```
Comment (modmuss50) [inline fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json]: You havent added this to the language builder.

Comment (modmuss50) [reply fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json]: I have pushed the fix.

Comment (TelepathicGrunt) [reply fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json]: Oh thank you, you beat me to it

**Review by modmuss50 — APPROVED** [review]

**Comment by modmuss50** [comment]
It still failed!

**Pull request merged**